### PR TITLE
Add openAPI changes

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

This is a sample PR showing the requested changes on open API tool.

### CODE REVIEW

File: openapi/thinkific/types.bal [hunk 1]
```diff
@@ -14,8 +14,79 @@
...
+    string host = "";
+    int port = 0;
+    string userName = "";
+    @display {
+        label: "",
+        kind: "password"
+    }
+    string password = "";
+|};
+
```
Comment (aneeshafedo) [inline openapi/thinkific/types.bal:89]: Missing `AuthConfig` record

File: openapi/api2pdf/client.bal [hunk 2]
```diff
@@ -40,8 +32,39 @@ public isolated client class Client {
     # + clientConfig - The configurations to be used when initializing the `connector`
```
Comment (aneeshafedo) [inline openapi/api2pdf/client.bal:32]:
```suggestion
    # + config - The configurations to be used when initializing the `connector` 
```

File: openapi/covid19/client.bal [hunk 3]
```diff
@@ -24,13 +24,42 @@ public isolated client class Client {
...
+                httpClientConfig.responseLimits = check config.responseLimits.ensureType(http:ResponseLimitConfigs);
+            }
+            if config.secureSocket is http:ClientSecureSocket {
+                httpClientConfig.secureSocket = check config.secureSocket.ensureType(http:ClientSecureSocket);
+            }
+            if config.proxy is http:ProxyConfig {
+                httpClientConfig.proxy = check config.proxy.ensureType(http:ProxyConfig);
+            }
+        }
+        self.clientEp = check new (serviceUrl, httpClientConfig);
```
Comment (aneeshafedo) [inline openapi/covid19/client.bal:62]:
```suggestion
        http:Client httpEp = check new (serviceUrl, httpClientConfig);
        self.clientEp = httpEp;
        return;
```

In other scenarios we have not directly assign values to `self.clientEp`. Shall we select one approach to maintain the consistency.

File: openapi/api2pdf/client.bal (hunk 2 above)
Comment (SkNuwanTissera) [reply openapi/api2pdf/client.bal]: Updated the PR

File: openapi/covid19/client.bal (hunk 3 above)
Comment (SkNuwanTissera) [reply openapi/covid19/client.bal]: Updated the PR

**Comment by LakshanSS** [comment]
@SkNuwanTissera Let's close this if this PR is not required anymore

**Comment by NipunaRanasinghe** [comment]
Closing as per the offline discussion I had with @LakshanSS, as we are working on cleaning up outdated PRs in the Ballerina platform repositories and this PR is not needed anymore.

**Pull request closed without merging**